src/components/Layouts/SidePanelsLayout.tsx

```diff
@@ -2,6 +2,10 @@ import { FC, ReactElement } from 'react';
 import { Box, styled, Theme } from '@material-ui/core';
 import { Breakpoint } from '@material-ui/core/styles/createBreakpoints';
 
+type ScrollBoxProps = {
+  isscroll?: 1 | 0;
```

Comment:
As I mentioned in the last scrum, `isScrollable` will face `React-DOM` issue until we use the latest version of styled-components. That's why I used all lowercased props and the other numbers rather than `boolean` type.

Reply:
If we use the latest version of styled-components, we can pass the props as `$isScrollable`.
Please refer to the [transient props](https://styled-components.com/docs/api#transient-props).